test/auth.test.js

```diff
@@ -81,6 +81,37 @@ describe('basic auth', () => {
     expect(await this.application.protocolService.getRootKey()).to.be.ok;
   }).timeout(20000);
 
+  it('cannot sign in twice to registered account', async function () {
```

Comment:
Can we also have a test that ensures that if a sign in fails (due to incorrect password), the `this.signingIn` mutex is correctly reset?